verl/workers/fsdp_workers.py

```diff
@@ -577,7 +575,7 @@ def compute_values(self, data: DataProto):
             load_fsdp_param_and_grad(module=self.critic_module,
                                      device_id=torch.cuda.current_device(),
                                      load_grad=self._is_offload_grad)
-        micro_batch_size = self.config.ppo_micro_batch_size
+        micro_batch_size = self.config.forward_micro_batch_size
```

Comment:
I wonder why we need an extra forward_micro_batch_size config?

Reply:
Sorry I miss your comments above. I think we can adopt larger micro batch size in fwd only computation as it doesn't require activation.

Reply:
Yes that's what I was thinking - fwd only computation can benefit from larger bs.